tmeister/static/src/toggles/table/feature-row/individual-toggle/individual-toggle.component.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import ToggleProdModal from '../../toggle-prod-modal.component.js'
+import { useChangeFeatureStatus } from '../../../toggles.hooks.js'
+
+export default function IndividualToggle (props) {
+  const { toggle, refetchToggles } = props
+  const [ toggleConfirmModal, setToggleConfirmModal ] = useState(false)
+  const [ setToggleToChange, setNewState ] = useChangeFeatureStatus(refetchToggles)
+  return (
+    <td key={toggle.toggle.env}>
```

Comment:
https://muppet.fandom.com/wiki/Hoggle

Reply:
NIT: maybe destructure down a layer to avoid the two frequent repetition of `toggle.toggle`

Reply:
Yeah, I started doing this but reverted my changes because I realized it was already _huge_ so I didn't want to lump another change into here.

I should have done this as multiple PRs.
1. Tabs -> Spaces
2. Remove Redux for create
3. Remove redux for toggles